### PR TITLE
[YUNIKORN-317] cache removal design doc

### PR DESCRIPTION
**Pull request by wilfred-s**
Adding the design doc for the cache removal implemented in YUNIKORN-317

Small side changes to clean up build and files:
Change to slug to define docs home
Removal of unused files (atom and doap)
Removal of unicode line separator chars (LSEP, U+2028)
Clean up of build script tabs

### CODE REVIEW

**Review by yangwwei — COMMENTED** [review]
hi @wilfred-s Thanks for the detailed write-up, this is great! What should we do with http://yunikorn.apache.org/docs/next/design/scheduler_core_design? I noticed you pointed to this doc in the old scheduler core design doc. To avoid confusion, is it better to merge this one into the old core design doc? We do not need it on the master branch for old reference, we can always go back to see in the older versions.

File: docs/design/cache_removal.md [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+ -->
+
+# Proposal to combine Cache and Scheduler's implementation in the core
+This document describes the current state of the scheduler and cache implementation.
+It describes the changes planned based on the analysis that was done of the current behaviour.
+
+## Goals
+The goal is to provide the same functionality before and after the change.
+- Junit tests before and after the merge must all pass.
+- Smoke tests defined in the core should all pass without major changes*.
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply docs/design/cache_removal.md]: the footnote link was not added correctly fixed that

File: docs/design/cache_removal.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+ * limitations under the License.
+ -->
+
+# Proposal to combine Cache and Scheduler's implementation in the core
+This document describes the current state of the scheduler and cache implementation.
+It describes the changes planned based on the analysis that was done of the current behaviour.
+
+## Goals
+The goal is to provide the same functionality before and after the change.
+- Junit tests before and after the merge must all pass.
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply docs/design/cache_removal.md]: yep that is a typo, fixed in the update

**Comment by wilfred-s** [comment]
The [core design doc](http://yunikorn.apache.org/docs/next/design/scheduler_core_design) needs a full rewrite. All that is there needs to be done from scratch. That is a good amount of work and will follow as a separate change. 

I expect that it will take at least 3 days to rewrite it all. There are a number of things in that doc that are not correct anymore.
When we have that done the existing document will be replaced. For now we just have a tag in it to show that it is not current.
Filed [YUNIKORN-495](https://issues.apache.org/jira/browse/YUNIKORN-495) for this

**Review by yangwwei — APPROVED** [review]
Sounds good, thanks!

**Comment by yangwwei** [comment]
@wilfred-s feel free to merge this with a proper commit message, thank you.

**Comment by wilfred-s** [comment]
commit pushed, auto build will publish soon.

**Pull request closed without merging**